pkg/features/kube_features.go

```diff
@@ -259,13 +259,6 @@ const (
 	// Lock to default and remove after v1.22 based on user feedback that should be reflected in KEP #1972 update
 	ExecProbeTimeout featuregate.Feature = "ExecProbeTimeout"
 
-	// owner: @jpbetz
-	// alpha: v1.30
-	// beta: v1.31
```

Comment:
Sorry one more thing, this file https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L189-L193 is missing the beta comment.

Reply:
@Jefftree, Okay!!

Reply:
Done.